lib/core/test/unit/Cardano/WalletSpec.hs

```diff
@@ -1258,17 +1258,18 @@ dummyTransactionLayer = TransactionLayer
                  , metadata = Nothing
                  , scriptValidity = Nothing
                  }
-        wit <- forM (view #inputs cs) $ \(_, TxOut addr _) -> do
-            (xprv, Passphrase pwd) <- withEither
-                (ErrKeyNotFoundForAddress addr) $ keystore addr
-            let sigData = tx ^. #txId . #getHash
-            let sig = CC.unXSignature $ CC.sign pwd (getKey xprv) sigData
-            return $ xpubToBytes (getKey $ publicKey xprv) <> sig
+        let wit = forMaybe (NE.toList $ view #inputs cs) $ \(_, TxOut addr _) -> do
+                (xprv, Passphrase pwd) <- keystore addr
+                let sigData = tx ^. #txId . #getHash
+                let sig = CC.unXSignature $ CC.sign pwd (getKey xprv) sigData
+                return $ xpubToBytes (getKey $ publicKey xprv) <> sig
```

Comment:
Not exactly relevant to this PR, but necessary because of the change of semantic when signing / the removal of the `ErrKeyNotFoundForAddress`